### PR TITLE
PBS PRO does not have a sync option.

### PR DESCRIPTION
**Pull request by johnjosephmorgan**
I am editing the abs.pl script to work with abs pro.

### CODE REVIEW

**Comment by danpovey** [comment]
Thanks.
Would you mind trying to resolve the conflicts?  You'd have to do it by merging with the Kaldi master and then resolving the conflict in pbs.pl by editing it (search for repeated equals signs) and then doing "git add" on that file, and git commit.

**Comment by johnjosephmorgan** [comment]
Yes, I'll try to do that.

File: egs/wsj/s5/utils/parallel/pbs.pl [hunk 1]
```diff
@@ -52,16 +52,12 @@
 # command qsub -v PATH -S /bin/bash -l arch=*64*
 # option mem=* -l mem_free=$0,ram_free=$0
 # option mem=0          # Do not add anything to qsub_opts
-# option num_threads=* -pe smp $0
```
Comment (danpovey) [inline egs/wsj/s5/utils/parallel/pbs.pl]: It's not safe to remove the num-threads option, many scripts use it.  PBS pro must have a mechanism to request more than one core per job.

**Comment by danpovey** [comment]
There are too many updates on this PR, so I'm closing it to avoid people subscribed to Kaldi getting spammed.  Please let me know when it's ready to review and I will re-open it.

**Comment by johnjosephmorgan** [comment]
Sorry. I have to figure out how to avoid this in the future.
It's not ready.
Thanks,

**Pull request closed without merging**